x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -246,6 +247,36 @@ export const serviceErrorGroupsRoute = createRoute({
   },
 });
 
+export const serviceThroughputRoute = createRoute({
+  endpoint: 'GET /api/apm/services/{serviceName}/throughput',
+  params: t.type({
+    path: t.type({
+      serviceName: t.string,
+    }),
+    query: t.intersection([
+      t.type({ transactionType: t.string }),
+      uiFiltersRt,
+      rangeRt,
+    ]),
+  }),
+  options: { tags: ['access:apm'] },
```

Comment:
Maybe we could add this option automatically for all APIs, it seems to be repeated everywhere. Not saying we should do it now.

Reply:
Hahah, it used to be like that but I changed it to be explicit :D